main.py

```diff
@@ -119,10 +120,11 @@ def main():
 
     if 'test' in datasets:
         test_loader = data_utils.get_dataset_loader(config, datasets['test'], model.word_dict, model.classes, train=False)
-        test_metrics = evaluate(model, test_loader, config.monitor_metrics)
+        test_metrics = evaluate(model, test_loader, config.monitor_metrics, config.silent)
```

Comment:
```suggestion
        test_metrics = evaluate(model, test_loader, config.monitor_metrics, silent=config.silent)
```
Should specify argument name, otherwise an error will be raised:
File "/home/sian/LibMultiLabel/libmultilabel/evaluate.py", line 24, in evaluate
    batch_labels = batch[label_key]
KeyError: True